src/main/java/org/jitsi/videobridge/octo/OctoEndpoint.java

```diff
@@ -153,4 +153,18 @@ void setReceiveSsrcs(Map<MediaType, Set<Long>> ssrcsByMediaType)
             ssrcs.forEach(ssrc -> streamInformationStore.addReceiveSsrc(ssrc, mediaType));
         });
     }
+
+    @Override
+    public boolean isSendingAudio()
+    {
+        // TODO implement detection
```

Comment:
Do you consider this TODO as needing to be resolved before this PR gets merged?

Reply:
No, I would like to get this merged, so we can start gathering data while we have octo turnedd off. Implementing this on top of Brian's transport refactor (and some more octo endpoint refactoring) would be cleaner, which is why I left it out for now.